index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Poppins"
+      rel="stylesheet"
+    />
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <nav>
+      <div id="logoContainer"><p id="logo">My Logo</p></div>
+      <div id="burgerContainer"><img id="burgerMenu" src="burger.png" /></div>
+    </nav>
+    <section id="headline">
+      <h1 id="greeting">I’m Andrew. Glad to see you!</h1>
+      <p id="intro">
+        I’m a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hesitate to contact me.
+      </p>
+      <p id="connect">LET'S CONNECT</p>
+      <div class="social">
+        <a href="https://www.twitter.com" target="_self"
+          ><img src="tw.png" alt="twitter icon"
+        /></a>
+        <a href="https://www.linkedin.com" target="_self"
+          ><img src="linkedin.png" alt="linkedin icon"
+        /></a>
+        <a href="https://www.medium.com" target="_self"
+          ><img src="medium.png" alt="medium icon"
+        /></a>
+        <a href="https://www.github.com" target="_self"
+          ><img src="github.png" alt="github icon"
+        /></a>
+        <a href="https://www.vectorstock.com" target="_self"
+          ><img src="v.png" alt="vector stock icon"
+        /></a>
+      </div>
```

Comment:
- [ ] It is important to make use of semantic tags as much as possible in our application and as such i will suggest rather than using the non-semantic `div` tag as container for the social media icons, you make use of the semantic `ul` tag and each link would be wrapped in an `li` tag as a list item. see the example below for more reference :+1: 
```html
<ul class="socials">
  <li>
    <a href="https://github.com/"><img src="./assets/github.png" alt="github"/></a>
  </li>
  <li>
    <a href="https://linkedin.com/"><img src="./assets/linkedin.png" alt="linkedin"/></a>
  </li>
</ul>
```